### DISCUSSION TITLE
Enable SSH without touch screen?

### DISCUSSION THREAD

**Discussion by Drizzt321** [mr-manuel/venus-os_dbus-serialbattery#114] [Q&A] [answered]
So just installed latest VenusOS on my RPi 2W, got it on my WiFi from the VictronConnect app, and have access to it via the GUI (both v1 & v2). I also have keyboard & mouse & monitor, so console shell access. I set a password via the shell, but I'm at a loss as to how to get SSH started up now. Doesn't appear to have `services` or `systemctl` installed, and the instructions at https://www.victronenergy.com/live/ccgx:root_access I can't use, because I have no touch screen so I can't "push and hold" the right button, so how am I supposed to get SSH access? A whole lot easier to be able to setup my BMS (or to use a PR version, since the LiTime hasn't been fully merged yet) via SSH from my main system where I can copy & paste stuff easily.

**Comment by mr-manuel** [comment, marked as answer]
If you only have local console access you can change the permissions via `dbus-spy`.

- Set `com.victronenergy.settings/Settings/System/AccessLevel`to `2` by selecting the entry, hitting enter, typing `2` and hit enter again.

- Set `com.victronenergy.settings/Settings/System/SSHLocal`to `1`.

OR

Since you have connected it to your WiFi, access via browser and keep dragged down the list for at least five seconds with your mouse on a PC or finger on a mobile device.

![venus-os-root-access](https://github.com/user-attachments/assets/517e57d0-5162-4ebd-8a36-0e24422b7c3f)

**Reply by Drizzt321** [reply]
Oh, that Gui v2 drag & hold worked. I thought it worked only in the Access level menu, thus only in the v1 GUI. Well that makes it easy... thanks!

Bed time now, but tomorrow time to figure out how to uninstall and then re-install from https://github.com/mr-manuel/venus-os_dbus-serialbattery/pull/59, since my battery is LiTime.
